experimental/python/wheel.bzl

```diff
@@ -242,6 +243,10 @@ to refer to the package in other packages' dependencies.
         "license": attr.string(default = ""),
         "classifiers": attr.string_list(),
         "description_file": attr.label(allow_single_file = True),
+        "package_root_path": attr.string(
```

Comment:
I'd suggest renaming this to something like "strip_path_prefix" to make it more explicit what happens here. In particular, I'd read package_root_path as path to prepend (there's a request to do that as well).

In addition, perhaps it would make sense to make this a list so that multiple prefixes to strip could be provided?

Reply:
Done

Reply:
Pay attention to the fact that the order of prefixes is important. I created separate tests to show that and documented it in the `wheelmaker` argument and hinted that in the rule attribute description.